data/map.txt

```diff
@@ -27975,6 +28192,15 @@ planet Mainsail
 		threshold 3500
 		fleet "Small Republic" 57
 
+planet Mahoganybox
+	attributes station uninhabited hai retirement
+	landscape land/space6
+	description `A visitor to this station will immediately notice that the gravity is lower than most planets by about a third. Leaving the docking bay brings one into an expansive room that can only be described as stately. A round reception desk greets you just inside, and is situated such that the occupants can face any direction that suits them and observe the room. A smiling Hai in a suit informs you that unless you have family at this respite facility, you will not be permitted to wander beyond this room. You are, however, welcome to visit with anyone who has come down or ask an orderly if you need to contact someone on business.`
```

Comment:
```suggestion
	description `A visitor to this station will immediately notice that the gravity is lower than most planets by about a third. Leaving the docking bay brings you into an lavishly decorated room with a donut-shaped reception desk made of an exotic material that you can't immediately identify. A smiling Hai in a suit behind the desk informs you that this is a respite home for the elderly, and unless you have family here, you will not be permitted to wander beyond this room. You are, however, welcome to visit with anyone who has come down or ask an orderly if you need to contact someone on business.`
```

Reply:
Stately and lavish have sufficiently different connotations that I'm not sure I'm on board with this particular suggestion as it stands now.

Reply:
If you're going for a wealthy genteel vibe, maybe change "lavishly decorated" in my suggestion into "tastefully decorated"?

Reply:
Yeah ok I'm down with that.
```suggestion
	description `A visitor to this station will immediately notice that the gravity is lower than most planets by about a third. Leaving the docking bay brings you into a tastefully decorated room with a donut-shaped reception desk made of an exotic material that you can't immediately identify. A smiling Hai in a suit behind the desk informs you that this is a respite home for the elderly, and unless you have family here, you will not be permitted to wander beyond this room. You are, however, welcome to visit with anyone who has come down or ask an orderly if you need to contact someone on business.`
```

Reply:
Cool, sounds good.